app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,5 +1,5 @@
 .home-header {
-  padding: ($base-spacing * 2) 0;
+  padding: 0 0 ($base-spacing * 2) 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 0 $base-spacing * 2` instead of `0 0 $base-spacing * 2 0`